release notes/v0.46.0.md

```diff
@@ -52,12 +96,20 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [#3159](https://github.com/grafana/pull/3159) Adds per-vu TLS configuration to `grpc.Client` connect method's parameters. Thanks @chrismoran-mica 🙇‍♂️.
 - [#3176](https://github.com/grafana/pull/3176) Adds a `js/promises` package, which enables extension developers to easily create promises that will be dispatched to the eventloop using the `New` function.
 - [#3181](https://github.com/grafana/pull/3181) Adds a `RunOnEventLoop` method to the `modulestest.Runtime` type, which allows extensions developers to run code on the event loop from their tests.
+- [browser#901](https://github.com/grafana/xk6-browser/pull/901) Adds support for shadow DOM piercing in `locator`. Thanks to @tmc for its implementation.
+- [browser#953](https://github.com/grafana/xk6-browser/pull/953) Improves Web Vitals reporting for better accuracy and consistency.
 
 ## Bug fixes
 
 - [#3163](https://github.com/grafana/pull/3163) Addresses a bug in our gRPC example.
 - [#3178](https://github.com/grafana/pull/3178) Fixes the registration of nested protobuf messages in the gRPC module.
 - [#3170](https://github.com/grafana/pull/3170) Upgrades the version of gRPC k6 depends on to address a security vulnerability ([CVE-2023-32731]).
+- [browser#866](https://github.com/grafana/xk6-browser/pull/866) Disables the timeout on browser process execution.
+- [browser#942](https://github.com/grafana/xk6-browser/pull/942) Fixes deadlock in `frameNavigated`.
+- [browser#943](https://github.com/grafana/xk6-browser/pull/943) Fixes a race condition in navigation and lifecycle events.
+- [browser#979](https://github.com/grafana/xk6-browser/pull/979) Fixes a deadlock on `pagehide` event evaluation when `page.close()` is called.
```

Comment:
```suggestion
- [browser#979](https://github.com/grafana/xk6-browser/pull/979) Temporary fix for a deadlock on `pagehide` event evaluation when `page.close()` is called.
```

Reply:
I'm not sure we should mention "temporary" in release notes, especially for something that is more internal. If the change fixes by now wouldn't it be better to just say "Fixes"?